components/brave_rewards/browser/extension_rewards_service_observer.h

```diff
@@ -8,6 +8,7 @@
 #include <memory>
 
 #include "base/macros.h"
+#include "bat/ledger/ledger.h"
```

Comment:
Without this inclusion, it seems there is no definition for `PublisherInfoList`, so if I remove this I get the following on build:

`error: no type named 'PublisherInfoList' in namespace 'ledger'; did you mean 'PublisherInfo'?`

Is there another resource I should use?

Reply:
strange, because we use ledger in this file already

Reply:
we definitely cannot have bat/ledger/* in a public brave_rewards header file and we shouldn't be exposing any ledger::* types either. There are rewards types which wrap them

Reply:
@bridiver understood. This ledger type was exposed in the file already:

https://github.com/ryanml/brave-core/blob/187381759b445a9682e79600a02fdc9830bd89ff/components/brave_rewards/browser/extension_rewards_service_observer.h#L36

That's why I thought it was strange that I needed to include anything at all. Where can I find these rewards types?

Reply:
content_site.h

Reply:
also ledger::PublisherInfo was exposed using a forward decl in RewardsServiceObserver, but OnGetPublisherActivityFromUrl also needs to change

Reply:
I also commented here OnGetPublisherActivityFromUrl https://github.com/brave/brave-core/pull/840/files#diff-d04cc8232108b15f3e9654dd88f79951R57 cc @emerick 
no ledger header files or types can be exposed to anything outside of components/brave_rewards

Reply:
@bridiver removed `bat/ledger` inclusion, and am now using `ContentSiteList`:

https://github.com/brave/brave-core/pull/957/commits/be4e3bcee46c4d4868f6c0f1e0d29ec7dff21d5e

I did not touch `OnGetPublisherActivityFromUrl` as that seems like it will be handled in a different PR